docker/entrypoint.d/50-gpu-driver-check2.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 
-if [[ "${NVIDIA_CPU_ONLY:-0}" == "1" ]]; then
```

Comment:
Don't change this file, it is correct for the non cpu-only triton image.
You need to copy this file into docker/cpu_only/entrypoint.d and then remove the condition as noted in the other comment.